src/catalogue/items/catalogueItemsLandingPage.component.tsx

```diff
@@ -422,65 +427,69 @@ function CatalogueItemsLandingPage() {
-                        />
-                      </ListItem>
+              {manufacturer && (
+                <Grid item xs={12}>
+                  <Collapse in={showManufacturer}>
+                    <Grid container spacing={1} sx={{ px: '192px' }}>
+                      <Grid item xs={12}>
+                        <Typography color="text.primary">Name</Typography>
+                        <Typography color="text.secondary">
+                          {manufacturer?.name}
+                        </Typography>
+                      </Grid>
+                      <Grid item xs={12}>
+                        {manufacturer?.url && (
+                          <Grid item xs={12}>
+                            <Typography color="text.primary">URL</Typography>
+                            <Typography sx={{ margin: '8px' }} variant="body1">
+                              <MuiLink
+                                underline="hover"
+                                href={manufacturer.url}
+                              >
+                                {manufacturer.url}
+                              </MuiLink>
+                            </Typography>
+                          </Grid>
+                        )}
+                      </Grid>
+                      <Grid item xs={12}>
+                        <Typography color="text.primary">
+                          Address Line
+                        </Typography>
+                        <Typography color="text.secondary">
+                          {manufacturer?.address.address_line}
+                        </Typography>
+                      </Grid>
+                      <Grid item xs={12}>
+                        <Typography color="text.primary">Town</Typography>
+                        <Typography color="text.secondary">
+                          {manufacturer?.address.town}
+                        </Typography>
+                      </Grid>
+                      <Grid item xs={12}>
+                        <Typography color="text.primary">County</Typography>
+                        <Typography color="text.secondary">
+                          {manufacturer?.address.county}
+                        </Typography>
+                      </Grid>
+                      <Grid item xs={12}>
+                        <Typography color="text.primary">Country</Typography>
+                        <Typography color="text.secondary">
+                          {manufacturer?.address.country}
+                        </Typography>
+                      </Grid>
+                      <Grid item xs={12}>
+                        <Typography color="text.primary">
+                          Post/Zip code
+                        </Typography>
+                        <Typography color="text.secondary">
+                          {manufacturer?.address.postcode}
+                        </Typography>
+                      </Grid>
```

Comment:
I think these probably need styles to match the formatting above:
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/f9df4bd4-c38f-4408-b800-69f4522ccaa2)